project/Settings.scala

```diff
@@ -98,13 +99,16 @@ object Settings {
     dockerUpdateLatest := true,
     dockerCommands ++= List(
       ExecCmd("CMD", "export", "CLASSPATH=lib/*jar")
+//      ExecCmd("CMD", "unmask", "umask g+w") //TODO: doesn't seem to do anything yet
     )
   )
 
   lazy val serverDockerSettings = commonDockerSettings ++ List(
     mainClass in Compile := Some("org.broadinstitute.dsp.workbench.welder.server.Main"),
     packageName in Docker := "broad-dsp-gcr-public/welder-server",
-//    daemonUser in Docker := "welder-user",
+    daemonUser in Docker := "welder-user",
+    daemonGroup in Docker := "user",
+    daemonGroupGid in Docker := Some("25000"),
```

Comment:
Should the gid be consistent with the Jupyter image? It's 100 there I think.

Reply:
yea, I was wondering if I should use gid in jupyter, but I didn't know about it...So just used a random one as stub